ng2-components/ng2-activiti-processlist/README.md

```diff
@@ -239,19 +240,42 @@ This is a sub-component of the process details component, which renders some gen
     processInstance="localProcessDetails">
 </adf-process-instance-details>
 ```
+![adf-process-instance-header](docs/assets/adf-process-instance-header-attachment.png)
 
 ### Properties
 
 | Name | Type| Description |
 | --- | --- | --- |
 | processInstance | string | (**required**): Full details of the process instance to display information about |
-| showDiagram | boolean | If the value is true the button show diagram is shown |
 
 ### Events
 
-| Name | Description |
-| --- | --- |
-| showProcessDiagram | Raised when the show diagram button is clicked |
+No events
+
+## ProcessInstanceModel
+
+```json
```

Comment:
https://github.com/Alfresco/alfresco-ng2-components/blob/master/ng2-components/ng2-activiti-processlist/src/models/process-instance.model.ts can we link it?